.changeset/few-cherries-attack.md

```diff
@@ -0,0 +1,5 @@
+---
+'ai': patch
+---
+
+fix (ai/core): resolve </think> edge cases in reasoning extractor middleware
```

Comment:
would rephrase (also PR title) to describe what functionality has been added, e.g. "add startWithReasoning option to extractReasoningMiddleware"